.appveyor.yml

```diff
@@ -14,34 +14,36 @@ matrix:
   fast_finish: true
 
 build_script:
-  - cmake --version
-  - SET PATH=%PATH%;"C:\Program Files\MySql\MySQL Server 5.7\bin"
+  - SET PATH=%PATH%;C:\Program Files\MySQL\MySQL Server 5.7\bin
   - set MYSQL_PWD=Password12!
   - SET
   - mysql -e "create database sqlpp_mysql;" --user=root
   - CD
   - cd ..
   - CD
-  - git clone https://github.com/MariaDB/mariadb-connector-c
-  - cd mariadb-connector-c
-  - mkdir build
-  - cd build
-  - cmake .. -DCMAKE_GENERATOR_PLATFORM=x64
-  - cmake --build . --config %configuration% --target install
-  - CD ..
-  - CD ..
-  - CD
   - git clone https://github.com/HowardHinnant/date
   - cd date
-  - git checkout tags/v2.4
-  - cd ..
-  - git clone https://github.com/rbock/sqlpp11
+  - git checkout tags/v3.0.1
+  - mkdir build && mkdir prefix && cd build
+  - cmake -DCMAKE_INSTALL_PREFIX="%cd%\..\prefix" -DBUILD_TZ_LIB=OFF ..
+  - cmake --build . --target install
+  - cd ..\..
+  - git clone https://github.com/kovdan01/sqlpp11.git
```

Comment:
I guess we'll have to wait for you PR in sqlpp11 to land?

Reply:
Ah, sorry, forgot about that. Temporarily cloned my own repository when used "Visual Studio 2013" worker image on appveyor - it only supports CMake 3.12.2, while sqlpp11 library requires CMake 3.14 or higher. Now "Visual Studio 2019" worker image with CMake 3.20.2 is used, so there is no need in using my fork. Changed to using 0.60 tag from original rbock/sqlpp11.